### DISCUSSION TITLE
Article on blumon

### DISCUSSION THREAD

**Discussion by potto216** [bluez/bluer#144] [Show and tell]
Hi All, I posted an article on blumon at https://medium.com/@potto_94870/watching-bluetooth-with-rust-using-bluers-command-line-tools-7028e696abf6 .  The article should be accessible by anyone without a Medium account. If not let me know.  
--Paul

**Comment by surban** [comment]
One comment: an easier way to install `blumon` than cloning the repository is to run `cargo install bluer-tools`. You can also use `cargo install` with git repositories directly, if you need an unreleased version.

**Reply by potto216** [reply]
Thanks, I didn't know that. I updated the story and shortened it--always a good thing:).
